packages/connector-core/src/types.ts

```diff
@@ -32,18 +32,18 @@ export type ConnectorMetadata = {
 };
 
 export enum ConnectorErrorCodes {
-  General = 'General',
-  InvalidMetadata = 'InvalidMetadata',
-  InvalidConfigGuard = 'InvalidConfigGuard',
-  InsufficientRequestParameters = 'InsufficientRequestParameters',
-  InvalidConfig = 'InvalidConfig',
-  InvalidResponse = 'InvalidResponse',
-  TemplateNotFound = 'TemplateNotFound',
-  NotImplemented = 'NotImplemented',
-  SocialAuthCodeInvalid = 'SocialAuthCodeInvalid',
-  SocialAccessTokenInvalid = 'SocialAccessTokenInvalid',
-  SocialIdTokenInvalid = 'SocialIdTokenInvalid',
-  AuthorizationFailed = 'AuthorizationFailed',
+  General = 'general',
+  InvalidMetadata = 'invalid_metadata',
+  InvalidConfigGuard = 'invalid_config_guard',
+  InsufficientRequestParameters = 'insufficient_request_parameters',
+  InvalidConfig = 'invalid_config',
+  InvalidResponse = 'invalid_response',
+  TemplateNotFound = 'template_not_found',
+  NotImplemented = 'not_implemented',
+  SocialAuthCodeInvalid = 'oauth_code_invalid',
+  SocialAccessTokenInvalid = 'invalid_access_token',
+  SocialIdTokenInvalid = 'invalid_id_token',
```

Comment:
should align with key? e.g. `social.auth_code_invalid` or `social_auth_code_invalid`